frontend/pages/home_page.js

```diff
@@ -49,16 +84,83 @@ export default class HomePage extends Component {
     );
   }
 
+  onChange = ({ name, value }) => {
+    this.setState({ [name]: value })
+  }
+
+  checkBoxOnChangeComplete = ({ name, value }) => {
+    // console.log('checkbox');
+    // console.log('name & value', name, value)
+    // if(this.state[name]) {
+    //   this.setState({ [name]: '' });
+    // } else {
+    //   this.setState({ [name]: value });
+    // }
+    if(this.state.complete) {
```

Comment:
i think all the checkboxes can use the same onChange method, also, you have to capture the onchange event differently. The reason `FormInput` `onChange` handles `({ name, value })` is because it wraps the low level logic. If you're using a plain `<input>` element, you'll need to handle the `onChange` differently. You can do:

```
onChangeChexbox = (e) => {
  // i think the field you wan it 'checked'
  const { name, checked } = e.target;
  this.setState({ [name]: checked });
};
```

the value of `checked` can vary based on browsers, some are 'true/false' others are 'checked' (I think). Either way, it doesnt matter, we can check for truthy/falsy when we check:

```
if (this.state.complete) ...
```